### PR TITLE
[bitnami/phpmyadmin] allows to create ServiceAccount resource

### PR DESCRIPTION
**Pull request by brainiac84**
Signed-off-by: Vadim Grek <vadimprog@gmail.com>

### Applicable issues

  - fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by brainiac84** [comment]
@javsalgar mind to check

**Comment by Mauraza** [comment]
Hi @brainiac84,

Could you tell us what are the benefits or reasons for this change?

**Comment by brainiac84** [comment]
Hi @Mauraza, I've been trying to use [Vault CSI Provider](https://developer.hashicorp.com/vault/docs/platform/k8s/csi#authenticating-with-vault) to get the database credentials from Hashicorp Vault and set them as environment variables, but in order to make auth to work it requires service account to exists and attached to the deployment.

File: bitnami/phpmyadmin/templates/serviceaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{{- if .Values.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
+metadata:
+  name: {{ template "phpmyadmin.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
```
Comment (Mauraza) [inline bitnami/phpmyadmin/templates/serviceaccount.yaml:7]: I think in this line you need to quotes
```suggestion
  namespace: {{ .Release.Namespace | quote }}
```

**Review by Mauraza — COMMENTED** [review]
Hi @brainiac84,

Could you check the comment, please?

**Review by Mauraza — APPROVED** [review]
Hi @brainiac84,

Thanks for the contribution! LGTM!

**Pull request merged**